src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -451,47 +483,40 @@ private static void EnsureValidPropertyType(string propertyName, Type propertyTy
         ///     If <c>true</c>, evaluates all the properties, otherwise just checks that
         ///     ones marked with <see cref="RequiredAttribute" /> are not null.
         /// </param>
-        /// <param name="breakOnFirstError">Whether to break on the first error or validate everything.</param>
         /// <returns>A list of <see cref="ValidationError" /> instances.</returns>
         private static IEnumerable<ValidationError> GetObjectPropertyValidationErrors(object instance,
-            ValidationContext validationContext, bool validateAllProperties, bool breakOnFirstError)
```

Comment:
If desired, this should be entirely separate change as it changes code, not just code-style.